target/linux/ipq40xx/image/generic.mk

```diff
@@ -276,7 +276,29 @@ define Device/avm_fritzbox-7530
 	SOC := qcom-ipq4019
 	DEVICE_PACKAGES := fritz-caldata fritz-tffs-nand
 endef
-TARGET_DEVICES += avm_fritzbox-7530
+
+define Device/avm_fritzbox-7530-toshiba
+	$(call Device/FitImageLzma)
+	DEVICE_VENDOR := AVM
+	DEVICE_MODEL := FRITZ!Box 7530 (Toshiba NAND)
```

Comment:
Using `DEVICE_VARIANT` instead for the NAND type feels more semantically correct.